tests/test-generate-docs.sh

```diff
@@ -24,9 +24,10 @@ rm -rf docs-test
 for TYPE in $(ls -d docs/*); do
   # Skip directories that are unpopulated:
   TYPE=`basename $TYPE`
-  LOOKFOR="docs/$TYPE/$TYPE-metadata.tsv"
-  if [ ! -e $LOOKFOR ]; then
-    echo "Skipping $TYPE. To add: touch $LOOKFOR"
+  LOOKFOR_ASSAY="docs/$TYPE/$TYPE-metadata.tsv"
+  LOOKFOR_OTHER="docs/$TYPE/$TYPE.tsv"
```

Comment:
Is there something to call this other than `OTHER` - are these entities?

Reply:
There probably could be a better name, but "other" is at least consistent: `get_other_schema`, `table-schemas/others`, and on the GH Pages homepage, `Other TSVs`. These are the TSVs that aren't _assay metadata_ TSVs: ie, contributors, antibodies, samples.

Reply:
Sounds good